cmd/eksctl-anywhere/cmd/upgradeplancluster.go

```diff
@@ -111,7 +112,9 @@ func (uc *upgradeClusterOptions) upgradePlanCluster(ctx context.Context) error {
 	componentChangeDiffs.Append(fluxupgrader.FluxChangeDiff(currentSpec, newClusterSpec))
 	componentChangeDiffs.Append(capiupgrader.CapiChangeDiff(currentSpec, newClusterSpec, deps.Provider))
 	componentChangeDiffs.Append(cilium.ChangeDiff(currentSpec, newClusterSpec))
-
+	componentChangeDiffs.Append(
+		eksdupgrader.ChangeDiff(cluster.NewManagementComponents(currentSpec.RootVersionsBundle().VersionsBundle), cluster.NewManagementComponents(newClusterSpec.RootVersionsBundle().VersionsBundle)),
```

Comment:
don't hate me
but does it make sense to use management components for an eksd diff?
eksd is not a management component. Is a cluster component

Reply:
Fair point. Yeah, this confusion arises in the messages we show in the diff as well

Maybe, we should just keep this one as is referencing the cluster spec